content/patterns/listbox/examples/listbox-collapsible.html

```diff
@@ -254,14 +267,13 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
             </tr>
             <tr data-test-id="listbox-aria-activedescendant">
               <td></td>
-              <th scope="row"><code>aria-activedescendant=&quot;ID_REF&quot;</code></th>
+              <th scope="row"><code>aria-activedescendant="ID_REF"</code></th>
               <td><code>ul</code></td>
               <td>
                 <ul>
-                  <li>Set by the JavaScript when it displays and sets focus on the listbox; otherwise is not present.</li>
-                  <li>Refers to the option in the listbox that is visually indicated as having keyboard focus.</li>
+                  <li>When an option in the listbox is visually indicated as having keyboard focus, refers to that option.</li>
+                  <li>Enables assistive technologies to know which element the application regards as focused while DOM focus remains on the listbox element.</li>
```

Comment:
```suggestion
                  <li>Enables assistive technologies to know which element the application regards to as focused while  the DOM focus remains on the listbox element.</li>
```

Reply:
Adding 'to' would change the meaning. Perhaps we could have used "is identifying" or "is indicating" instead of "regards". As I rethink the wording, it seems that use of "regards" is somewhat anthropomorphizing the web page by implying that the web page can hold something in regard. I don't want to change the wording in this one case, however, because we use identical wording on several example pages. We could consider a PR that revises all instances of this phrase if this is a problematic wording. I'm not convinced that it is.